.github/workflows/build-v84.yml

```
@@ -0,0 +1,125 @@
+        id: meta
+        if: ${{ steps.image_exists.outcome == 'failure' }}
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # ${{ env.CURRENT_VERSION }}
+          tags: |
+            type=raw,value=${{ env.CURRENT_VERSION }}
+
+      # Install Docker Buildx Plugin
+      - name: Install Docker Buildx Plugin
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC1083:warning:2:54: This { is literal. Check expression (missing ;/\n?) or quote it

(shellcheck)